flake8_routable.py

```diff
@@ -354,6 +356,18 @@ def rename_migrations(self) -> None:
                 reported.add(line_token.start[0])
                 self.errors.append((*line_token.start, ROU109))
 
+    def disallow_called_prefix_in_unit_tests(self) -> None:
+        """.called_, .has_calls, .not_called should not be called in mocks"""
+        reported = set()
+
+        for token in self._file_tokens:
+            token_str = token.string
+            if token.type == tokenize.NAME and (
+                token_str.startswith("called_") or token_str in {"has_calls", "not_called"}
+            ):
+                reported.add(token.start[0])
+                self.errors.append((*token.start, ROU114))
```

Comment:
Missing pieces:

- Scope to only Mocks
- Scope to only test files

Reply:
From our conversation in slack with Justin and Tom, we're ok with this simple approach until we upgrade to 3.12